lib/hammer_cli_foreman_tasks/recurring_logic.rb

```diff
@@ -27,6 +27,20 @@ class CancelCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class ClearCancelledCommand < HammerCLIForeman::DeleteCommand
+      def request_params
+        params = super
+        params["search"] = "state=cancelled"
```

Comment:
Could there be cases when someone might want to delete other logics than cancelled?

My thinking is that API does not force the predefined search param, so we could delete other logics using API, not just those that are cancelled.

Reply:
I'm not sure if we want to add this command as a bulk destroy command or a clear cancelled command? @adamruzicka

Reply:
How about we add a couple of predefined searches so people can do something like `hammer recurring-logic clear --cancelled` and `hammer recurring-logic clear --finished`?

I'm a bit afraid that if we let the users to provide any search at all, they would break things

Reply:
So we want the command to have `finished, cancelled and search` options or just ` finished and cancelled` ?

Reply:
I'd vote for `finished` and `cancelled`

Reply:
Added these options